src/test/java/core/basesyntax/FileReaderTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.impl.FileReadServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderTest {
+    private static FileReader fileReader;
+
+    @BeforeClass
+    public static void before() {
+        fileReader = new FileReadServiceImpl();
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void invalidPath_Not_Ok() {
```

Comment:
lets change all test names and follow pattern 
`<method name>_<desription>_<expected behavior>`